modules/pxp-module-puppet.bat

```diff
@@ -3,4 +3,4 @@ SETLOCAL
 
 call "%~dp0..\..\bin\environment.bat" %0 %*
 
-ruby -S -- "%~dp0%SCRIPT_NAME%" %*
+ruby -S -- "%~dp0\pxp-module-puppet" %*

```

Comment:
For some reason SCRIPT_NAME wouldn't resolve to anything. For the other batch files we are installing for puppet-agent we have also just started hard coding the name. Is there a reason to keep SCRIPT_NAME and not change it?